_sass/_base.scss

```diff
@@ -563,6 +563,9 @@ a.btn-targets {
     // padding: 8px 15px;
     font-size: 12px;
   }
+  @include phone{
+    margin-left:10px !important;
```

Comment:
Are the importants here really necessary?  I usually try to avoid using important unless it's a helper class.  I find that I can using improve my selector to avoid needing importants.

http://www.smashingmagazine.com/2010/11/02/the-important-css-declaration-how-and-when-to-use-it/